window.go

```diff
@@ -226,9 +226,15 @@ func (c *Window) ProcessEvent(ev Event) bool {
 		}
 		return true
 	case EventKey:
-		if ev.Key == term.KeyTab {
+		if ev.Key == term.KeyTab || ev.Key == term.KeyArrowUp || ev.Key == term.KeyArrowDown {
```

Comment:
Window is the first control that receives an event. If we apply the patch,, it means that no Controls receives arrow keys because Window consumed them. Have you tried your fix with Controls like ListBox, TextView or TableView?

Maybe make a small change:
```
229  if ev.Key == term.KeyTab || ev.Key == term.KeyArrowUp || ev.Key == term.KeyArrowDown {
230      if SendEventToChild(c, ev) {
231                  return true
232      }
233 nextControl := true
234 # your other code goes here
```
?